dvc/command/plot.py

```diff
@@ -0,0 +1,228 @@
+        "show",
+        parents=[parent_parser],
+        description=append_doc_link(SHOW_HELP, "plot/show"),
+        help=SHOW_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    plot_show_parser.add_argument(
+        "-t",
+        "--template",
+        nargs="?",
+        default=None,
+        help="File to be injected with data.",
+    )
+    plot_show_parser.add_argument(
+        "datafile", nargs="?", default=None, help="Data to be visualized."
+    )
+    plot_show_parser.add_argument(
+        "-r", "--result", help="Name of the generated file."
+    )
+    plot_show_parser.add_argument(
+        "--no-html",
+        action="store_true",
+        default=False,
+        help="Do not wrap vega plot json with HTML.",
+    )
+    plot_show_parser.add_argument(
+        "-f",
+        "--fields",
+        default=None,
+        help="Choose which fileds or jsonpath to put into plot.",
+    )
+    plot_show_parser.add_argument(
+        "-o",
+        "--stdout",
+        action="store_true",
+        default=False,
+        help="Print plot content to stdout",
+    )
+    plot_show_parser.add_argument(
+        "--no-csv-header",
+        action="store_true",
+        default=False,
+        help="Provided csv datafile does not have a header.",
+    )
+    plot_show_parser.set_defaults(func=CmdPlotShow)
+
+    PLOT_DIFF_HELP = (
+        "Plot changes in metrics between commits"
+        " in the DVC repository, or between a commit and the workspace."
+    )
+    plot_diff_parser = plot_subparsers.add_parser(
+        "diff",
+        parents=[parent_parser],
+        description=append_doc_link(PLOT_DIFF_HELP, "plot/diff"),
+        help=PLOT_DIFF_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    plot_diff_parser.add_argument(
+        "-t",
+        "--template",
+        nargs="?",
+        default=None,
+        help=("File to be injected with data."),
+    )
+    plot_diff_parser.add_argument(
+        "-d",
+        "--datafile",
+        nargs="?",
+        default=None,
+        help="Data to be visualized.",
+    )
+    plot_diff_parser.add_argument(
+        "-r", "--result", help="Name of the generated file."
+    )
+    plot_diff_parser.add_argument(
+        "revisions",
+        nargs="*",
+        default=None,
+        help="Git revisions to plot from",
+    )
+
+    plot_diff_parser.add_argument(
+        "--no-html",
+        action="store_true",
+        default=False,
+        help="Do not wrap vega plot json with HTML.",
+    )
+    plot_diff_parser.add_argument(
+        "-f",
+        "--fields",
+        default=None,
+        help="Choose which filed(s) or jsonpath to put into plot.",
+    )
+    plot_diff_parser.add_argument(
+        "-o",
+        "--stdout",
+        action="store_true",
+        default=False,
+        help="Print plot content to stdout",
+    )
+    plot_diff_parser.add_argument(
+        "--no-csv-header",
```

Comment:
@dmpetrov @pared can we name it just `--no-header`? simpler and covers TSV case.

Reply:
I agree here, @dmpetrov ?